server/enclave/CMakeLists.txt

```diff
@@ -28,9 +28,6 @@ set_target_properties(enclave PROPERTIES
 target_compile_definitions(enclave
   PRIVATE
   OE_API_VERSION=2)
-if($ENV{OE_SIMULATION})
```

Comment:
This is being deleted because building the enclave executable actually does not need to know if simulation is enabled or not, only the host cmake does